dune_client/api/extensions.py

```diff
@@ -118,6 +118,27 @@ def get_latest_result(
         except KeyError as err:
             raise DuneError(response_json, "ResultsResponse", err) from err
 
+    def get_latest_result_dataframe(
```

Comment:
yes, this is a performance improvement for the examples on https://github.com/duneanalytics/dune-client/pull/99/files